eo-maven-plugin/src/main/java/org/eolang/maven/DcsWithRuntime.java

```diff
@@ -23,47 +23,107 @@
  */
 package org.eolang.maven;
 
+import com.jcabi.xml.XMLDocument;
+import java.io.IOException;
+import java.net.URL;
 import java.util.Iterator;
 import org.apache.maven.model.Dependency;
 import org.cactoos.list.ListOf;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.scalar.Unchecked;
 
 /**
  * Add runtime dependency if it is absent.
  *
  * @since 0.28.11
- * @todo #1361:90min Hardcoded version of EoRuntimeDependency.
- *  See the EoRuntimeDependency constructor for more info.
- *  It's much better to determine the version of the runtime library
- *  dynamically. For example, we can fetch the latest version by http
- *  or from config files.
  */
 final class DcsWithRuntime implements Dependencies {
 
     /**
-     * The dependencies source.
+     * Dependency downloaded by HTTP from Maven Central.
+     */
+    private static final Unchecked<Dependency> FROM_MAVEN = fromMaven();
+
+    /**
+     * All dependencies.
      */
     private final Dependencies delegate;
 
     /**
-     * The main constructor.
+     * Runtime dependency source.
+     */
+    private final Unchecked<Dependency> source;
+
+    /**
+     * Constructor.
      *
-     * @param delegate Dependencies source
+     * @param delegate Dependencies delegate.
      */
     DcsWithRuntime(final Dependencies delegate) {
+        this(delegate, DcsWithRuntime.FROM_MAVEN);
+    }
+
+    /**
+     * The main constructor.
+     *
+     * @param delegate Dependencies delegate.
+     * @param source Runtime dependency source.
+     */
+    DcsWithRuntime(final Dependencies delegate, final Unchecked<Dependency> source) {
         this.delegate = delegate;
+        this.source = source;
     }
 
     @Override
     public Iterator<Dependency> iterator() {
         final ListOf<Dependency> all = new ListOf<>(this.delegate);
         if (all.stream().noneMatch(new RuntimeDependencyEquality())) {
-            final Dependency dependency = new Dependency();
-            dependency.setGroupId("org.eolang");
-            dependency.setArtifactId("eo-runtime");
-            dependency.setVersion("0.28.10");
-            dependency.setClassifier("");
-            all.add(dependency);
+            all.add(this.source.value());
         }
         return all.iterator();
     }
+
+    /**
+     * Runtime dependency source from Maven Central.
+     *
+     * @return Runtime dependency from Maven Central.
+     */
+    private static Unchecked<Dependency> fromMaven() {
```

Comment:
@volodya-lombrozo let's rename to `download`

Reply:
@mximp I don't think it's a good name for that static method according with that [post](https://www.yegor256.com/2018/08/22/builders-and-manipulators.html).

Reply:
@mximp renamed to `mavenDependency`